packages/react-examples/src/react/DetailsList/DetailsList.CustomGroupHeaders.Example.tsx

```diff
@@ -55,26 +65,43 @@ export class DetailsListCustomGroupHeadersExample extends React.Component<{}, {}
 
     this._items = createListItems(500);
     this._groups = createGroups(GROUP_COUNT, 1, 0, ITEMS_PER_GROUP);
+    this.state = { selectedItem: dropdownOptions[0] };
   }
 
   public render(): JSX.Element {
     return (
-      <DetailsList
-        items={this._items}
-        groups={this._groups}
-        groupProps={{
-          onRenderHeader: this._onRenderGroupHeader,
-          onRenderFooter: this._onRenderGroupFooter,
-        }}
-        getGroupHeight={this._getGroupHeight}
-        ariaLabelForSelectionColumn="Toggle selection"
-        ariaLabelForSelectAllCheckbox="Toggle selection for all items"
-        checkButtonAriaLabel="select row"
-        onRenderDetailsHeader={this._onRenderDetailsHeader}
-      />
+      <>
+        <Dropdown
+          label="Checkbox visibility"
+          selectedKey={this.state.selectedItem.key}
+          // eslint-disable-next-line react/jsx-no-bind
+          onChange={this._onChange}
+          placeholder="Select checkbox visibility"
```

Comment:
You shouldn't need the lint disable, and the placeholder won't do anything since there's always a selected option.
```suggestion
          onChange={this._onChange}
```